builtin/providers/profitbricks/data_source_datacenter_test.go

```diff
@@ -0,0 +1,31 @@
+package profitbricks
+
+import (
+	"github.com/hashicorp/terraform/helper/resource"
+	"regexp"
+	"testing"
+)
+
+func TestAccDataSourceDatacenter_basic(t *testing.T) {
```

Comment:
I tried something similar: 

In first step I do this:

```
resource "profitbricks_datacenter" "foobar" {
	name       =  "datacenter-test"
	location = "us/las"
}
```
and then this.
```
resource "profitbricks_datacenter" "foobar" {
	name       =  "datacenter-test-updated"
	location = "us/las"
}

data "profitbricks_datacenter" "data_source"{
	name = "datacenter-test-updated"
	location= "us/las"
}
```

Each time I do this it simply doesn't get work. 
How do you want me to proceed?